salt/tests/unit/modules/test_metalk8s_volumes.py

```diff
@@ -435,38 +435,25 @@ def test_clean_up(self, name, raise_msg=False, pillar_volumes=None,
                 # This function does not return anything
                 metalk8s_volumes.clean_up(name)
 
+
     @parameterized.expand([
-        param('my-device'),
-        param(exists_return=False, raises=True, result='device `/dev/my-device` not found'),
-        param(exists_return=[False, False, False, True], result='my-device')
+        # Nominal case: device exists.
+        ([True], 'my-device'),
+        # Error case: device doesn't exists.
+        ([False]*10, 'device `/dev/my-device` not found'),
+        # Device is temporarily missing.
+        ([False, False, False, True], 'my-device'),
```

Comment:
FWIW, you can use a string as first argument to each test case, it will be appended to the generated function name:

```python
    @parameterized.expand([
        ("exists", [True], 'my-device'),
        ("does not exist", [False]*10, 'device `/dev/my-device` not found'),
        ("temporarily missing", [False, False, False, True], 'my-device'),
    ])
    def test_device_name(self, _, exist_values, result):
        # ...
```

So instead of getting `pytest` output similar to:

```
salt/tests/unit/modules/test_metalk8s_volumes.py::Metalk8sVolumesTestCase::test_device_name_0 PASSED [ 33%]
salt/tests/unit/modules/test_metalk8s_volumes.py::Metalk8sVolumesTestCase::test_device_name_1 PASSED  [ 66%]
salt/tests/unit/modules/test_metalk8s_volumes.py::Metalk8sVolumesTestCase::test_device_name_2 PASSED  [100%]
```

You'll have:


```
salt/tests/unit/modules/test_metalk8s_volumes.py::Metalk8sVolumesTestCase::test_device_name_0_exists PASSED [ 33%]
salt/tests/unit/modules/test_metalk8s_volumes.py::Metalk8sVolumesTestCase::test_device_name_1_does_not_exist PASSED  [ 66%]
salt/tests/unit/modules/test_metalk8s_volumes.py::Metalk8sVolumesTestCase::test_device_name_2_temporarily_missing PASSED  [100%]
```

Reply:
Done